packages/mjml-core/src/helpers/preview.js

```diff
@@ -7,5 +7,8 @@ export default function (content) {
     <div style="display:none;font-size:1px;color:#ffffff;line-height:1px;max-height:0px;max-width:0px;opacity:0;overflow:hidden;">
       ${content}
     </div>
+    <div style="display:none;font-size:1px;color:#ffffff;line-height:1px;max-height:0px;max-width:0px;opacity:0;overflow:hidden;">
```

Comment:
This was copied from line 7. We've been using `display: none; max-height: 0px; overflow: hidden;` from the article so not sure about the differences between these two hide styles